sdk/eventgrid/azure-eventgrid/swagger/postprocess_eventnames.py

```diff
@@ -0,0 +1,23 @@
+import inspect
+import re
+from azure.eventgrid._generated import models
+
+def event_tuples(system_events):
+    tup_list = []
+    for event in system_events:
+        class_name = event[0].replace("EventData", "EventName")
+        try:
+            event_name = re.findall("Microsoft.[a-zA-Z]+.[a-zA-Z]+", event[1].__doc__)[0]
+        except:
+            print(event[0])
```

Comment:
We should be more explicit here that we weren't able to generate the mapping and sys.exit(1) at least to be sure any automation that would run this script would be stopped and blocked.